relational_operators/SelectOperator.hpp

```diff
@@ -87,7 +89,26 @@ class SelectOperator : public RelationalOperator {
         num_workorders_generated_(0),
         simple_projection_(false),
         input_relation_is_stored_(input_relation_is_stored),
-        started_(false) {}
+        placement_scheme_(input_relation.getNUMAPlacementScheme()),
```

Comment:
@adalbertgerald This method is valid only if `QUICKSTEP_HAVE_LIBNUMA`. Therefore, we need to use `#ifdef` for this statement as well.